packages/manager/modules/office/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@ovh-ux/office",
+  "version": "7.0.8",
+  "description": "OVH Microsoft Office module",
+  "keywords": [
+    "angular",
+    "microsoft"
+  ],
+  "repository": "@ovh-ux/office",
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
```

Comment:
Missing both **browser** and **module** entries.

Reply:
I used rollup to check if the build was successful but i think we can remove it now and only use src/index.js as a main entry point. Are we ok with that @jleveugle @cbourgois @marie-j ?

Reply:
Nop, this module will only provide a main entry targeting the src/index.js at this time.

Reply:
Must be also reflected to the rollup.config.js file then.

Reply:
`rollup.config.js` is now removed :)

Reply:
Ok got it! Thanks!